docs/blog/pydantic-v2.md

```diff
@@ -0,0 +1,945 @@
+
+For years I've wanted to clean up the model namespace,
+see [pydantic#1001](https://github.com/samuelcolvin/pydantic/issues/1001). This would avoid confusing gotchas when field
+names clash with methods on a model, it would also make it safer to add more methods to a model without risking
+new clashes.
+
+After much deliberation (and even giving a lightning talk at the python language submit about alternatives, see
+[this discussion](https://discuss.python.org/t/better-fields-access-and-allowing-a-new-character-at-the-start-of-identifiers/14529)).
+I've decided to go with the simplest and clearest approach, at the expense of a bit more typing:
+
+All methods on models will start with `model_`, fields' names will not be allowed to start with `"model"`
+(aliases can be used if required).
+
+This will mean `BaseModel` will have roughly the following signature.
+
+```{.py .annotate title="New BaseModel methods"}
+class BaseModel:
+    model_fields: List[FieldInfo]
+    """previously `__fields__`, although the format will change a lot"""
+    @classmethod
+    def model_validate(cls, data: Any, *, context=None) -> Self:  # (1)
+        """
+        previously `parse_obj()`, validate data
+        """
+    @classmethod
+    def model_validate_json(
+        cls,
+        data: str | bytes | bytearray,
+        *,
+        context=None
+    ) -> Self:
+        """
+        previously `parse_raw(..., content_type='application/json')`
+        validate data from JSON
+        """
+    @classmethod
+    def model_is_instance(cls, data: Any, *, context=None) -> bool: # (2)
+        """
+        new, check if data is value for the model
+        """
+    @classmethod
+    def model_is_instance_json(
+        cls,
+        data: str | bytes | bytearray,
+        *,
+        context=None
+    ) -> bool:
+        """
+        Same as `model_is_instance`, but from JSON
+        """
+    def model_dump(
+        self,
+        include: ... = None,
+        exclude: ... = None,
+        by_alias: bool = False,
+        exclude_unset: bool = False,
+        exclude_defaults: bool = False,
+        exclude_none: bool = False,
+        mode: Literal['unchanged', 'dicts', 'json-compliant'] = 'unchanged',
+        converter: Callable[[Any], Any] | None = None
+    ) -> Any:
+        """
+        previously `dict()`, as before
+        with new `mode` argument
+        """
+    def model_dump_json(self, ...) -> str:
+        """
+        previously `json()`, arguments as above
+        effectively equivalent to `json.dump(self.model_dump(..., mode='json'))`,
+        but more performant
+        """
+    def model_json_schema(self, ...) -> dict[str, Any]:
+        """
+        previously `schema()`, arguments roughly as before
+        JSON schema as a dict
+        """
+    def model_update_forward_refs(self) -> None:
+        """
+        previously `update_forward_refs()`, update forward references
+        """
+    def model_copy(self, ...) -> Self:
```

Comment:
Glad it's live. You forgot to remove this

Reply:
Good catch, will do so later.

Reply:
While I'm at it. There are some typos on `TypedDict` and you use the `full` name instead of `total`